src/views/checkups/storage/components/CheckupsStorageActions.tsx

```diff
@@ -24,6 +24,13 @@ const CheckupsStorageActions = ({
 }) => {
   const { t } = useKubevirtTranslation();
   const navigate = useNavigate();
+
+  const [newestJob] = jobs
+    .filter((it) => it?.metadata?.creationTimestamp)
+    .sort((a, b) => a.metadata.creationTimestamp.localeCompare(b.metadata.creationTimestamp))
+    .reverse();
```

Comment:
why reverse? why not flipping a, b in sort?

Reply:
It's more readable this way. Similarly it's considered beneficial to express conditional functions as "positives".
If you prefer I can add inline negation (multiply by `-1`).

Reply:
i dont think its morereadibly, and changing to b.metadata.creationTimestamp.localeCompare(a.metadata.creationTimestamp)) , is readble + save another run over the array.